optaplanner-test/pom.xml

```diff
@@ -29,25 +29,6 @@
       <artifactId>optaplanner-core</artifactId>
     </dependency>
     <!-- External dependencies -->
-    <!-- Common utils -->
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-lang3</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-    </dependency>
-    <!-- Logging -->
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-classic</artifactId>
```

Comment:
we'll need tests for the ScoreVerifier I fear, so we might need logging for those, no?